MANIFEST.in

```diff
@@ -21,6 +21,8 @@ include README.md
 recursive-include superset/examples *
 recursive-include superset/migrations *
 
+include superset/assets/package.json
```

Comment:
So we're back to serving up `package.json`?

Reply:
It's not in `static/assets` but in `assets/` instead which isn't served. Oh and at packaging time symlinks get materialized.